Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
+    elsif (var/"postgres-#{version_suffix}").exist?
+      puts ""
```

Comment:
Current situation works:

```
~> ls -1 /usr/local/var | grep postgres
postgres
postgres-9.4

We have not attempted to migrate your data due to your multi-postgresql setup.
```

The other two elements function as expected as well.